ios_widget_catalog_compare/pubspec.yaml

```diff
@@ -27,7 +27,7 @@ dependencies:
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^1.0.0
```

Comment:
we should leave it (so that if we eventually create a cupertino_icons 2.0 and break the apis, we don't accidentally auto-upgrade and break people).